test/Azure.Functions.Cli.Tests/ActionsTests/ResolveActionTests.cs

```diff
@@ -22,6 +22,7 @@ public class ResolveActionTests
         [InlineData("azure functionapp enable-git-repo appName", typeof(DeprecatedAzureActions))]
         [InlineData("azure functionapp fetch-app-settings appName", typeof(FetchAppSettingsAction))]
         [InlineData("azure functionapp fetch appName", typeof(FetchAppSettingsAction))]
+        [InlineData("azure functionapp publish app-name -g resource-group", typeof(FetchAppSettingsAction))]
```

Comment:
The test change is not validating the code change (the exit code is non zero). Can we add/update the tests do so?

Reply:
I added such a test below; do you mean something else?